restaurant/models/restaurant.py

```diff
@@ -0,0 +1,65 @@
+from django.core.exceptions import ValidationError
+from django.core.validators import MinValueValidator
+
+from django.db import models
+
+
+class RestaurantType(models.Model):
+
+    """This class is using for creating table with types of Restaurants"""
+
+    rest_type = models.CharField(max_length=256, blank=False)
+    is_deleted = models.BooleanField(default=False)
+
+    class Meta(object):
+
+        """Correct displaying of restaurant type in restaurant's list"""
+
+        verbose_name=u"Restaurant type"
```

Comment:
Please, fix this according to PEP 8.

> Always surround these binary operators with a single space on either side: assignment ( = ), augmented assignment ( += , -= etc.), comparisons ( == , < , > , != , <> , <= , >= , in , not in , is , is not ), Booleans ( and , or , not ).
[PEP 8](https://www.python.org/dev/peps/pep-0008/#other-recommendations)